framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -306,10 +306,25 @@
     private final Map<Tree, AnnotatedTypeMirror> classAndMethodTreeCache;
 
     /**
-     * Mapping from a Tree to its annotated type; before implicits are applied, just what the
-     * programmer wrote.
+     * Mapping from an expression tree to its annotated type; before implicits are applied, just
+     * what the programmer wrote. Might alias the same object as fromMemberTreeCache and
```

Comment:
The maps might alias each other?  I'm not sure what this last sentence means.

Reply:
The previous situation was that we had one field and one cache.
This changes it to three fields and three caches. This comment was supposed to say that we could also have three fields that all use the same cache.
I think we should just stick with the three separate caches and I'll remove the comment.